1. El segundo catch no ocurriria nunca, dado a que mas arriba se valida una expecion mas generica (Entra siempre al catch de arriba).
2. Si el numero es par, arrojaria una excepcion y retornaria 3 por el finally.
	Si el numero es impar el metodo devuelve 3.
(falta abrir llave en la linea del catch)